Podcast/TabbedPageViewController.swift

```diff
@@ -174,31 +146,10 @@ class TabbedPageViewController: UIViewController, UIPageViewControllerDataSource
     
     func updateSearchResults(for searchController: UISearchController) {
         guard let searchText = searchController.searchBar.text else { return }
-        
-        if searchText == "" {
-            view.addSubview(pastSearchesTableView)
-            view.bringSubview(toFront: pastSearchesTableView)
-            return
-        }
-        
-        if pastSearchesTableView.superview == view {
-            pastSearchesTableView.removeFromSuperview()
-        }
-    
-        if let timer = searchDelayTimer {
-            timer.invalidate()
-            searchDelayTimer = nil
-        }
-        
-        searchDelayBlock = {
-            self.searchText = searchText
-            self.sectionOffsets = [.episodes: 0, .series: 0, .people: 0, .tags: 0]
-            //TODO: @drew @mindy -> comment this block to be .people or .series
-            self.fetchData(type: .all, query: searchText, offset: 0, max: self.pageSize)
-            searchController.searchResultsController?.view.isHidden = false
-        }
-        
-        searchDelayTimer = Timer.scheduledTimer(timeInterval: 0.5, target: self, selector: #selector(searchAfterDelay), userInfo: nil, repeats: false)
```

Comment:
Why are you getting rid of the timer? This is how we rate limit our searches as they type.

Reply:
Whoops! will add it back in with a delay of .25 because .5 seemed a bit slow